src/python/WMCore/WMSpec/WMStep.py

```diff
@@ -50,12 +50,24 @@ def getNumberOfCores(self):
 
         Return the number of cores for the step in question
         """
-
         try:
             return int(self.data.application.multicore.numberOfCores)
-        except:
+        except Exception:
             return 1
 
+    def getNumberOfStreams(self):
+        """
+        _getNumberOfStreams_
+
+        Return the number of event streams for the step in question
+        """
+        # FIXME: I'm not really sure it's actually used
+        print("HEY, GET OUT!!!")
```

Comment:
Yes, it looks like we use this one in the unit tests. I have to remove this comment/print